MQ2PluginManager.cpp

```diff
@@ -79,9 +79,17 @@ static void DrawGUI()
 	ImGui::TextColored(ImVec4(0.0f, 1.0f, 1.0f, 1.0f), "/pluginman");
 	ImGui::SameLine();
 	ImGui::Text("Toggles the GUI window.");
+
+	char searchPlugin[MAX_STRING] = { 0 };
```

Comment:
I’m confused how this works given this variable will be empty every frame. Also are you expecting a plugin name that will be 2048 characters?

Reply:
shrank that to 24 chars long. should be able to find one without needing the entire name but this should cover entire name for all current plugins.